tools/dds/dds-config/eth-config-header.h

```diff
@@ -0,0 +1,18 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+#pragma once
+
+#include <cstdint>
+
+
+#pragma pack( push, 1 )
+
+// The header structure for eth config
+typedef struct {
```

Comment:
Why do we typedef this struct and not others?

Reply:
Changed